order.go

```diff
@@ -46,17 +46,41 @@ type OrderUpdateParams struct {
 	Status                 OrderStatus
 }
 
+// OrderReturnParams is the set of parameters that can be used when returning
+// orders. For more details, see: https://stripe.com/docs/api#return_order.
+type OrderReturnParams struct {
+	Params
+	Items []*OrderItemParams
```

Comment:
You'd know better than I, but is there a fundamental link between an order return item and an order item that's guaranteed to stay true in the future? You'd know better than I, but there's some possibility that you may just want to define a new `OrderReturnItemParams` struct if there's some chance that they'll deviate down the line.